client/sections.js

```diff
@@ -7,8 +7,7 @@ const path = require( 'path' );
 /**
  * Internal dependencies
  */
-const config = require( 'config' );
```

Comment:
I'm not sure if this is a good idea, i may put `config` back.  I think i removed it while i was looking into why `config` was breaking everything